src/LiveDevelopment/LiveDevelopment.js

```diff
@@ -868,6 +873,11 @@ define(function LiveDevelopment(require, exports, module) {
             var closeDeferred = (brackets.platform === "mac") ? NativeApp.closeLiveBrowser() : $.Deferred().resolve();
             closeDeferred.done(function () {
                 _setStatus(STATUS_INACTIVE, reason || "explicit_close");
+                // clean-up registered servers
+                _regServers.forEach(function (server) {
+                    LiveDevServerManager.removeServer(server);
```

Comment:
Yes, that would be good since the server keeps running after closing the session. Let me know if you've created the issue, I can take a look and submit a PR for that.